src/components/utils/validateQuestionInputs.js

```diff
@@ -0,0 +1,45 @@
+export const validateQuestionInputs = (inputs) => {
+  let errors = []
+
+  inputs.alternatives.forEach(({ correct, text }, index) => {
+    if (correct && text.length <= 4) {
+      errors.push(`A resposta não está preenchida`)
+    } else if (text.length <= 4) {
+      errors.push(`A ${index + 1}º alternativa incorreta não está preenchida`)
+    }
+  })
+
+  if (inputs.authorshipYear.length === 0) {
```

Comment:
Caraca.. validações no front is old school

Reply:
sad, mas por enquanto vai ser o jeito 👀

Reply:
na verdade tudo bem ter mas tá vamos ter que usar muito amarrado por enquanto